# Fixed shutdown after user inactivity in case a plugin is keeping the OSD open (cont'd).
# 
--- vdr.c	2018/07/16 08:52:40	5.2
+++ vdr.c	2019/03/12 10:01:16
@@ -1514,7 +1514,7 @@
               ShutdownHandler.countdown.Cancel();
            }
 
-        if (!cRecordControls::Active() && !RecordingsHandler.Active() && (Now - cRemote::LastActivity()) > ACTIVITYTIMEOUT) {
+        if (!cControl::Control() && !cRecordControls::Active() && !RecordingsHandler.Active() && (Now - cRemote::LastActivity()) > ACTIVITYTIMEOUT) {
            // Shutdown:
            // Check whether VDR will be ready for shutdown in SHUTDOWNWAIT seconds:
            time_t Soon = Now + SHUTDOWNWAIT;
